src/styles/blocks/articles.css

```
@@ -12,6 +12,110 @@
     font-size: 21px;
 }
 
+@media screen and (max-width: 320px) {
+
+    .main{
+        box-sizing: border-box;
+        display: grid;
+    }
+
+
+    h1{
+        position: relative;
+
+        font-family: RF Dewi;
```

Не указано семейство шрифта и кавычки:
```suggestion
        font-family: 'RF Dewi', sans-serif;
```